storage/src/cache/fscache/mod.rs

```diff
@@ -266,13 +273,20 @@ impl FileCacheEntry {
         )?));
         Self::restore_chunk_map(blob_info.clone(), file.clone(), &meta, &chunk_map);
 
+        let mut blob_data_file_path = String::new();
+        if cas_mgr.is_some() {
+            blob_data_file_path = get_path_from_file(&file);
```

Comment:
`get_path_from_file` is not recovered in tests. `cas_mgr` is empty?

[coverage(https://app.codecov.io/gh/dragonflyoss/nydus/pull/1507/blob/storage/src/utils.rs)